### DISCUSSION TITLE
Daily 26/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#231] [General]
Ayer estuve bastante ocupado en el trabajo pero pude empezar a incorporar MySQL dentro del sistema por si fuese obligatorio. Todavía no probé pero ya tengo la creación de la base de datos si no existe y empecé a incorporar algunos queries. Voy a seguir con lo de MySQL aunque me estoy empezando a trabar porque hay queries que no están terminados y subidos al archivo de sql.

**Comment by NataliaAlvarezIspc** [comment]
Ayer no pude avanzar mucho.
Hoy intentare terminar todas las tareas que tengo pendientes de repositorio. Además estare estudiando para la evaluación de Full.

**Comment by Brian-Almada** [comment]
Ayer no pude tocar nada, solo la corrección de una issue realizando un nuevo commit. Hoy también se me complicó, recién ahora estoy viendo cómo resolver las tareas que me quedan.

**Comment by AhumadaCeleste** [comment]
Ayer estudie para el examen de front. Imprimir y  leí un poco del ultimo modulo de python.  Hoy participe de la clase de ingles y complete la lección de modo imperativo. Continúe estudiando para el examen próximo. Mañana quisiera resolver una dificultad que tengo para incorporar imágenes es wordPress, lo que supongo que esta ligado a alguna configuración por lo que investigue. Resolviendo eso ya terminaría el ejercicio practico individual e iniciaría con lo correspondiente a Almoneda.
